variables-and-mutability/src/main.rs

```
@@ -1 +1,51 @@
+the compiler warning about the variable being unused.
+
+Remove the underscore. Provide a compiler directive
+to silence the compiler warning about the variable
+being unused.
+*/
+
+const TOUCHDOWN_POINTS: i32 = 6;
+
+fn main() {
+    let season: &str = "Summer";
+    let mut points_scored: i32 = 35;
```

Clippy (1.86.0)

[warning] value assigned to `points_scored` is never read

(warning)